resources/log4j2.xml

```diff
@@ -2,7 +2,7 @@
 <Configuration>
   <Appenders>
     <Console name="STDOUT" target="SYSTEM_OUT" follow="true">
-      <PatternLayout pattern="%date %level %logger{2} :: %message%n%throwable">
+      <PatternLayout pattern="%date %level %logger{2} | %message %mdc%n%throwable">
```

Comment:
`%mdc` is the _. It is printed like `{a=1 b=2}` in the log line.

`|` is 1 character shorter than `::` 🤷‍♂️